back/mail-processing/idea_notifications.py

```diff
@@ -0,0 +1,59 @@
+from datetime import date, datetime
+from datetime import timedelta
+import logging
+from utils.common import progressive_chunks, SEND_BATCH_EMAIL_CHUNK_SIZE
+from utils.models import UserModel, IdeaModel, CommentModel, LikeModel, one
+from mail_templates.idea_notifications.send_idea_notifications import send_idea_notifications_bulk
+import os
+import sentry_sdk
+from sentry_sdk.integrations.aws_lambda import AwsLambdaIntegration
+import itertools
+from collections import defaultdict
+import json
+
+# logging.basicConfig()
+# log = logging.getLogger("pynamodb")
+# log.setLevel(logging.DEBUG)
+# log.propagate = True
+
+sentry_sdk.init(dsn=os.environ.get('SENTRY_DSN'), integrations=[AwsLambdaIntegration()])
+
+now = datetime.now()
+dayAgo = datetime.now() - timedelta(days=1)
+
+def last_comments():
+  comments_iterator = CommentModel.scan(CommentModel.createdDate >= dayAgo)
```

Comment:
scan is really expensive in dynamodb, e.g. it will read all comments and than filter than. So it will be really expensive. We need to use index

Reply:
Ok, I'll try index

Reply:
Looks like there is just no way to do this https://stackoverflow.com/a/38790120/1753349 unless I create a fake hash field with the same value, and use that for query, querying just by date looks like not possible in dynamodb